src/index.ts

```diff
@@ -109,13 +109,14 @@ export async function listen (handle: http.RequestListener, opts: Partial<Listen
     await clipboardy.write(url).catch(() => { opts.clipboard = false })
   }
 
-  const showURL = () => {
+  const showURL = (overrides?: Pick<ListenOptions, 'baseURL'>) => {
```

Comment:
- For cases like (https://github.com/nuxt/framework/pull/5337#issuecomment-1151067202) the initial banner differs from the subsequent one. 

- I think generally this a nice feature but maybe we can define `ShowURLOptions` with it's own type? We could also support `name` option to show a message like `Local (API): http://localhost:300/api`